src/esp/geo/OBB.cpp

```diff
@@ -107,7 +107,7 @@ OBB computeGravityAlignedMOBB(const vec3f& gravity,
     const float dd = d0[0] * d1[1] - d0[1] * d1[0];
 
     const float dx = s1[0] - s0[0];
-    const float dy = s1[1] - s0[0];
+    const float dy = s1[1] - s0[1];
```

Comment:
@Skylion007, can you remind me what was the bug here?

Reply:
dy was not actually calculating a delta y.